<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MpptTab</class>
 <widget class="QWidget" name="MpptTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="mpptAlive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Array Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>MPPT Number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="batteryVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="mpptNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>3</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="arrayCurrent">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="temperature">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="arrayVoltage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Battery Voltage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_31">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Array Current</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
